src/privatesend/privatesend.h

```diff
@@ -23,6 +23,9 @@ static const int PRIVATESEND_AUTO_TIMEOUT_MAX = 15;
 static const int PRIVATESEND_QUEUE_TIMEOUT = 30;
 static const int PRIVATESEND_SIGNING_TIMEOUT = 15;
 
+// Used to automatically adjust the number of participants in a round.
+static const int PRIVATESEND_QUEUE_GOAL_TIME = 40;
```

Comment:
Can't we just re-use `PRIVATESEND_QUEUE_TIMEOUT`?

Reply:
Privatesend queue timeout refers to the time since the last successful step. This refers to the total time. Maybe it could be reused but that'd depend on how long a PS queue normally takes

Reply:
Hmm, good point, agree.